plugins/linux/ifplugin/descriptor/interface.go

```diff
@@ -260,6 +270,22 @@ func (d *InterfaceDescriptor) Validate(key string, linuxIf *interfaces.Interface
 		if linuxIf.GetVeth().GetPeerIfName() == "" {
 			return kvs.NewInvalidValueError(ErrVETHWithoutPeer, "peer_if_name")
 		}
+	case *interfaces.Interface_Loop:
+		nsCtx := nslinuxcalls.NewNamespaceMgmtCtx()
+		nsHandle, err := d.nsPlugin.GetNamespaceHandle(nsCtx, linuxIf.Namespace)
+		if err != nil {
+			return kvs.NewInvalidValueError(ErrNamespaceDoesntExists, "namespace")
```

Comment:
This can be misleading. `GetNamespaceHandle()` also tries to create the namespace if missing (calls `getOrCreateNs()` inside) and returned error can mean that the namespace failed to be created or read, or something else.

Perhaps the namespace check can be moved to the `getLoopbackAlias` and use some error like 'LoopbackValidationError' for all cases (failed get/create namespace, failed switch to namespace, alias already exists) and log the actual case.